.devcontainer/devcontainer.json

```
@@ -0,0 +1,32 @@
+// For format details, see https://aka.ms/devcontainer.json
+{
+	"name": "pallets/flask",
+	"image": "mcr.microsoft.com/vscode/devcontainers/python:3",
```

```suggestion
	"image": "mcr.microsoft.com/devcontainers/python:3",
```

Can we use the [updated image path](https://github.com/devcontainers/images/tree/main/src/python)? Thanks!